README.md

```diff
@@ -5,39 +5,391 @@
+1. [Docker Quick Start Guide](#docker-quick-start-guide)
+2. [Standalone Mode Guide](#standalone-mode-guide)
 
-- [Docker Engine](https://docs.docker.com/engine/) 1.27+
-- Makefile toolchain
-- Unix-based operating system (e.g. Debian, Arch, Mac OS X)
+### Docker Quick Start Guide
 
-_NB: There is no compatibility with Windows environments at this time._
+Running the app with Docker allows for minimal installation and a quick setup. This is recommended **for evaluation use-cases only**, such as local development builds.
 
-#### Setup for Docker-only
+**TL;DR Steps:**
 
 1. Copy `.env-api.sample` as `.env-api` and `.env-issuer.sample` as `.env-issuer`. Please see the [configuration](#configuration) section for more details.
 2. Run `make up`. This launches 3 containers with Postgres, Redis and Vault. Ignore the warnings about variables, since those are set up in the next step.
 3. **If you are on an Apple Silicon chip (e.g. M1/M2), run `make run-arm`**. Otherwise, run `make run`. This starts up the issuer API, whose frontend can be accessed via the browser (default <http://localhost:3001>).
-4. Follow the [steps](#adding-ethereum-private-key-to-the-vault) for adding an Ethereum private key to the Vault.
-5. Follow the [steps](#creating-the-issuer-did) for creating an identity as your issuer DID.
-6. _(Optional)_ To run the UI with its own API, first copy `.env-ui.sample` as `.env-ui`. Please see the [configuration](#configuration) section for more details.
+4. Follow the [steps](#import-wallet-private-key-to-vault) for adding an Ethereum private key to the Vault.
+5. Follow the [steps](#create-issuer-did) for creating an identity as your issuer DID.
+6. _(Optional)_ To run the UI with its own API, first copy `.env-ui.sample` as `.env-ui`. Please see the [configuration](#development-ui) section for more details.
 7. _(Optional)_ Run `make run-ui` (or `make run-ui-arm` on Apple Silicon) to have the Web UI available on <http://localhost:8088> (in production mode). Its HTTP auth credentials are set in `.env-ui`. The UI API also has a frontend for API documentation (default <http://localhost:3002>).
 
-> If you want to run the UI app in development mode, i.e. with HMR enabled, please follow the steps in the [Development (UI)](#development-ui) section.
+#### Docker Guide Requirements
+
+- Unix-based operating system (e.g. Debian, Arch, Mac OS)
+- [Docker Engine](https://docs.docker.com/engine/) `1.27+`
+- Makefile toolchain `GNU Make 3.81`
+
+> _**NOTE:** There is no compatibility with Windows environments at this time._
+
+To help expedite a lot of the docker commands, many have been abstracted using `make` commands.
+Included in each step are the equivalent docker commands to show what is being run.
+
+#### Create Docker Configuration Files
+
+Make a copy of the following environment variables files:
+
+```bash
+# FROM: ./
+
+cp .env-api.sample .env-api;
+cp .env-issuer.sample .env-issuer;
+# (Optional - For Issuer UI)
```

Comment:
This can't be "optional" here, and "required" [later](https://github.com/0xPolygonID/sh-id-platform/pull/327/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R280).